pkg/apis/compliance/v1alpha1/compliancecheckresult_types.go

```diff
@@ -85,7 +86,8 @@ type ComplianceCheckResult struct {
 	// The severity of a check status
 	Severity ComplianceCheckResultSeverity `json:"severity"`
 	// A human-readable check description, what and why it does
-	Description string `json:"description,omitempty"`
+	ValueUsed   []string `json:"valuesUsed,omitempty"`
+	Description string   `json:"description,omitempty"`
```

Comment:
the comment for the description got lost and is now on top of `ValueUsed`, let's keep it on top of description as that comment is used to generate the CRD.

Reply:
I have fixed that and added comments to the `ValuesUsed` for descriptions